features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/publisher.commons.js

```diff
@@ -124,8 +124,35 @@ $(document).ready(function() {
 		$(this).next().click();
 	});
 
-	$('input[type="file"]').change(function() {
-		$(this).closest('.tbl-upload').find('.txt-filepath').val($(this).val());
+	$('input[type="file"]').change(function(evt) {
+        var acceptFileType = $(this).attr('acceptType');
```

Comment:
Code Review: It is better to use standard "accept" attribute. Ref: form-field.hbs line 47